api/query/query.go

```diff
@@ -14,9 +14,23 @@ import (
 	"github.com/web-platform-tests/wpt.fyi/shared"
 )
 
-// summary is the golang type for the JSON format in pass/total summary files.
-type summary map[string][]int
+// SummaryResult is the format of the data from summary files generated with the newest aggregation method.
+type SummaryResult struct {
+	// Status represents the 1-2 character abbreviation for the status of the test.
+	Status string `json:"s"`
+	// Counts represents the subtest counts (passes and total).
+	Counts []int `json:"c"`
+}
 
+// summary is the golang type for the JSON format in pass/total summary files.
+// It has an old structure and a new structure - each which represent summary files
+// that match the old or new summary format.
+type summary struct {
+	// Old summary format. This will be null if the summary is using the new format.
+	Old map[string][]int
+	// New summary format that keeps status information.
+	New map[string]SummaryResult
```

Comment:
I don't recommend using the naming of `New`. It is too close to the new in variable initialization

Reply:
That's a good idea - changed to `oldFormat` and `newFormat`